rpcs3/Emu/Cell/Modules/cellSaveData.cpp

```diff
@@ -610,6 +610,12 @@ static NEVER_INLINE s32 savedata_op(ppu_thread& ppu, u32 operation, u32 version,
 		//	//return CELL_OK;
 		//}
 
+		if (operation == SAVEDATA_OP_AUTO_LOAD && !fs::is_dir(dir_path))
```

Comment:
SAVE vs LOAD wasn't supposed to have any effect on function behaviour except visual.

Reply:
You're right cellSaveDataAutoLoad2 has the same exact behavior as cellSaveDataAutoSave2.
I'll investigate a bit more.